kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/PartitionedTableId.java

```diff
@@ -220,6 +244,14 @@ private static String dateToDayPartition(LocalDate localDate) {
       return localDate.format(DateTimeFormatter.BASIC_ISO_DATE);
     }
 
+    private String dateToMonthPartition(LocalDate localDate) {
+      return localDate.format(DateTimeFormatter.ofPattern("yyyyMM"));
```

Comment:
Any BigQuery docs you can reference here on this syntax?

Reply:
I assume is this one:
https://cloud.google.com/bigquery/docs/partitioned-tables

![image](https://user-images.githubusercontent.com/1346073/102631153-b03ce480-414d-11eb-97a0-04aca3c2cfb2.png)

Reply:
👍    thanks @robertotena